resources/lang/epo.ini

```diff
@@ -0,0 +1,114 @@
+error.noclipboard = "Ne ekzistas tondejo - kreu tondejon unue"
+warning.differentlevel = "[AVERTO] Vi redaktas mondon kiu vi ne enestas!"
+; commands
+command.info.title = "Informo"
+command.limit.current = "Nuntempa limo: {%0}"
+command.limit.set = "Block change limit was set to {%0}"
+command.limit.set = "Blokŝanĝlimo fariĝis {%0}"
+command.setrange.current = "Nuntempa atingopovo: {%0}"
+command.setrange.set = "Iloatingo fariĝis {%0}"
+command.biomeinfo.attarget = "Biomedio ĉe celo"
+command.biomeinfo.atposition = "Biomedio ĉe pozicio"
+command.biomeinfo.result = "{%0} bimedioj trovitaj en elektaro"
+command.biomeinfo.result.line = "ID: {%0} Nomo: {%1}"
+command.biomelist.title = "Biomedio listo"
+command.biomelist.result.line = "ID: {%0} Nomo: {%1}"
+command.brushname.set = "Brosonomo fariĝis \"{%0}\""
+command.clearclipboard.cleared = "Tondejoj forigitaj"
+command.flip.try = "Provante renversi tondejo per {%0}"
+command.flip.success = "Tondejo renversita"
+command.rotate.try = "Provante turnigi tondejon per {%0} da gradoj"
+command.rotate.success = "Tondejo turnigita"
+command.history.cleared = "Historio forigita"
+command.listchunks.found = "{%0} blokegoj found in selection"
+command.size = "Elektaro grandeco"
+; selection
+selection.pos1.set = "Pozicio 1 fariĝis X: {%0} Y: {%1} Z: {%2}"
+selection.pos2.set = "Pozicio 2 fariĝis X: {%0} Y: {%1} Z: {%2}"
+; session
+session.undo.none = "Neniu por malfari"
+session.undo.left = "Restas{%0} da malfaroagoj"
+session.redo.none = "Neniu por refari"
+session.redo.left = "Restas {%0} da refaroagoj"
+session.brush.added = "Aldonis {%0} al sesio"
+session.brush.deleted = "Foriĝis {%0} (UUID {%1})"
+session.brush.removed = "Formovis {%0} (UUID {%1})"
+session.language.set = "Agordis lingvon al {%0}"
+session.language.notfound = "Lingvo {%0} ne trovita, uzante defaŭlton"
+; task
+task.copy.success = "Sukcesis neinsinkrona kopio, daŭris {%0}, kopiis {%1} da blokoj el{%2}."
+task.count.success = "Sukcesis neinsinkrona analizo, daŭris {%0}"
+task.count.result = "{%0} da blokoj trovitaj el totalo de {%1} blocks"
+task.replace.success = "Sukcesis neinsinkrona enspacigado, daŭris {%0}, ŝanĝis {%1} da blokoj el{%2}."
+task.replace.success = "Sukcesis neinsinkrona interŝanĝo, daŭris {%0}, ŝanĝis {%1} da blokoj el{%2}."
+task.revert.undo.success = "Sukcesis neinsinkrona malfarado, daŭris {%0}, ŝanĝis {%1} da blokoj el{%2}."
+task.revert.redo.success = "Sukcesis neinsinkrona refarado, daŭris {%0}, ŝanĝis {%1} da blokoj el{%2}."
+; flags
+flags.keepexistingblocks = "Konservu jamaj blokoj"
+flags.keepair = "Konservu aero"
+flags.hollow = "Kava"
+flags.hollowclosed = "Kava kun ŝtopitaj ekstremaĵoj"
+flags.natural = "Natura"
+; tools
+; wand tool
+tool.wand = "Sorĉbastono"
+tool.wand.lore.1 = "Alklaku maldekstren por agordi la pozicio 1 de elektaro"
+tool.wand.lore.2 = "Alklaku dekstren por agordi la pozicio 2 de elektaro"
+tool.wand.lore.3 = "Use //togglewand to toggle it's functionality"
```

Comment:
Here is still a missing one ^^

Reply:
\*sigh\*
I double checked the file, so that *should* be everything now.